controllers/courses.js

```diff
@@ -620,99 +622,122 @@ router.get('/:courseId/', (req, res, next) => {
 	];
 
 	// ########################### start requests to new Editor #########################
+	let editorBackendIsAlive = true;
 	if (EDITOR_URL) {
-		promises.push(apiEditor(req).get(`course/${req.params.courseId}/lessons`));
+		promises.push(apiEditor(req)
+			.get(`course/${req.params.courseId}/lessons`)
+			.catch((err) => {
+				logger.warn('Can not fetch new editor lessons.', err);
+				editorBackendIsAlive = false;
+				return {
+					total: 0,
+					data: [],
+				};
+			}));
 	}
 
 	// ############################ end requests to new Editor ##########################
+	const [
+		course,
+		_lessons,
```

Comment:
lint: no underscore?

Reply:
It is old code too. In this PR i add only editorBackendIsAlive check and resort the error handling.